deny.toml

```diff
@@ -0,0 +1,71 @@
+# cargo-deny configuration for enterprise compliance
+
+[licenses]
+# List of explicitly allowed licenses
+allow = [
+    "MIT",
+    "Apache-2.0",
+    "Apache-2.0 WITH LLVM-exception",
+    "BSD-2-Clause",
+    "BSD-3-Clause",
+    "ISC",
+    "Unicode-DFS-2016",
+]
+
+# Confidence threshold for detecting licenses
+confidence-threshold = 0.9
+
+[[licenses.clarify]]
+name = "encoding_rs"
+expression = "(Apache-2.0 OR MIT) AND BSD-3-Clause"
+license-files = [
+    { path = "COPYRIGHT", hash = 0x39f8ad31 }
+]
+
+[bans]
+# Lint level for when multiple versions of the same crate are detected
+multiple-versions = "warn"
+
+# Deny specific crates
+deny = [
+    # Crates with known security issues
+    { name = "openssl", version = "<0.10.38" },
```

Comment:
can this be sourced from some external database instead? rustsec's advisory database is depended on below; is this even necessary?